modules/util/portal-tools-rest-builder/src/main/resources/com/liferay/portal/tools/rest/builder/dependencies/dto.ftl

```diff
@@ -142,7 +143,11 @@ public class ${schemaName} <#if dtoParentClassName?has_content>extends ${dtoPare
 		</#if>
```

Comment:
What about modifing the max?

Reply:
I could do it, I don't know if there is any expected error for that but I don't mind to change it

Reply:
If that is the error with the minimum functionality, then we must apply the change also to the maximum

Reply:
Just curious, do we have the error for Double parameters or it is not reproducible for that type?

Reply:
I will check that :grin:

Reply:
but if there is code for supporting the Double, we should check if it works; if there is no double in Liferay's REST Builder APIs, maybe we could test it by just including a new field in an existing API and validating it